ops/scripts/pipeline/check-deploy-bicep.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Is there any way to kick off the bicep deployment manually? There might be a scenario where the Azure environment got changed without any bicep changes.

Reply:
I added a bicep deployment flag that defaults to false and evaluates within the script

Reply:
Recommend adding some details about this in the Note section of the pull request. To serve as a warning in manual updates are made to resources in Azure directly.

Reply:
Do you mean the PR template?